test/parallel/test-stream-finished.js

```diff
@@ -630,3 +631,9 @@ testClosed((opts) => new Writable({ write() {}, ...opts }));
     }));
   }));
 }
+
+{
+  // This case is needed to support the send module
```

Comment:
This comment is a bit confusing because it sounds like `send` is an internal module. Explaining what the test is checking for would be more useful than referencing a 3rd party module on npm.

Reply:
updated, good choice.